.github/workflows/CI.yaml

```diff
@@ -29,8 +29,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.8, 3.9, "3.10"]
-        rdkit-version: ["2021.09", "2021.03", "2018.03.4"]
+        python-version: [3.7]
+        rdkit-version: ["2018.03.4"]
```

Comment:
I'd rather we kept the latest rdkit as our gold standard (even if this means things break for now).  I think I'd prefer this matrix to be `[2018, 2021]` to cover what used to work and what should work respectively.  Then whatever the py needs to be to get those two builds working